currency_list.go

```diff
@@ -0,0 +1,170 @@
+package sleet
+
+var CURRENCIES = map[Code]Currency{
```

Comment:
Is this list going to be used across all gateways? We need to check if all gateways follow the same ISO standard. For example Adyen differs from ISO on decimal precision for some currencies: https://docs.adyen.com/development-resources/currency-codes#note
Also it represents Serbian dinar as CSD and ISO represents it as RSD.

Reply:
Actually that is a good question and something I realized I should probably adjust in the Adyen code

They have their own representation stored in adyen-api-go/amount.go which I should be using. So the question is more - how do we as Bolt/sleet want to represent it? So this I just copied from hail

Reply:
Also how did you find that so quickly haha

Reply:
Haha, I was previously reading some Adyen docs and came across the note which says they differ from ISO. Then I just diff a diff between our list and theirs and saw a few differences, lol.

Ya, I think it makes sense to make changes in the Adyen code, but we'll have to make sure we do corresponding changes in other gateways if they differ. Braintree seems fine though.

Reply:
Yeah definitely noted - hmm then do you have an opinion on the stance we should take with Sleet?

Reply:
Ya I think this PR does the right thing, we need a generic standard. If a gateway has some differences from ISO standard, we can write a gateway specific currency/amount adjustment logic.